From edaa40430041b53eab55e45b30877d55248fb108 Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Wed, 22 Jun 2016 10:10:16 +0900
Subject: [PATCH] unset push.followTags when pushing in `dgit push`

Otherwise, the push to dgit-repos might fail because git will push lots
of tags that the dgit-repos post-receive hook will reject.
---
 dgit | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dgit b/dgit
index e02a6bd..6182ddc 100755
--- a/dgit
+++ b/dgit
@@ -2061,7 +2061,7 @@ END
     if (!check_for_git()) {
 	create_remote_git_repo();
     }
-    runcmd_ordryrun @git, qw(push),access_giturl(),
+    runcmd_ordryrun @git, qw(-c push.followTags=false push),access_giturl(),
         $forceflag."HEAD:".rrref(), $forceflag."refs/tags/$tag";
     runcmd_ordryrun @git, qw(update-ref -m), 'dgit push', lrref(), 'HEAD';
 
-- 
2.8.1

